packages/tables/resources/views/index.blade.php

```diff
@@ -658,7 +658,7 @@ class="filament-tables-record-action-button flex-1 block py-3"
                                 :alignment="$column->getAlignment()"
                                 :sortable="$column->isSortable() && (! $isReordering)"
                                 :sort-direction="$getSortDirection()"
-                                :class="$getHiddenClasses($column)"
+                                class="filament-table-cell-{{ $column->getName() }} {{ $getHiddenClasses($column) }}"
```

Comment:
Might want to convert column name to kebab case?

Reply:
> Might want to convert column name to kebab case?
We can do this. 

> I feel like header and search input should have different class from regular cells.
Header cells should be easy to target via `th.filament-table-cell-...`. For search we could add an additional class.

Example use case: Add a border to certain column.

Reply:
Thats fine, what still needs doing to merge this?

Reply:
Converted the column name to kebab case. Unfortunately `str()->kebab()` doesn't work on snake case directly, so we need to use `str()->camel()->kebab()`